src/renderer/modals/RepairModal/index.js

```diff
@@ -106,36 +106,11 @@ const FlashStep = ({
   );
 
 const ErrorStep = ({ error }: { error: Error }) => (
-  <Box>
-    <Container>
-      <Box color="alertRed">
-        <ExclamationCircleThin size={44} />
-      </Box>
-      <Box
-        color="palette.text.shade100"
-        mt={4}
-        fontSize={6}
-        ff="Inter|Regular"
-        textAlign="center"
-        style={{ maxWidth: 350 }}
-      >
-        <TranslatedError error={error} field="title" />
-      </Box>
-      <Box
-        color="palette.text.shade80"
-        mt={3}
-        fontSize={4}
-        ff="Inter"
-        textAlign="center"
-        style={{ maxWidth: 380 }}
-      >
-        <TranslatedError error={error} field="description" />
-        <ol style={{ textAlign: "justify" }}>
-          <TranslatedError error={error} field="list" />
-        </ol>
-      </Box>
-    </Container>
-  </Box>
+  <ErrorDisplay error={error} withExportLogs>
+    <ol style={{ textAlign: "justify" }}>
```

Comment:
to me this should be an "in-place" feature inside `ErrorDisplay` to also do `<TranslatedError error={error} field="list" />` for you. probably only if it exists tho. to be checked what we can but to me it makes sense this happen for all error screens.